src/test/java/com/glencoesoftware/bioformats2raw/test/ZarrTest.java

```diff
@@ -868,6 +869,65 @@ public void testHCSMetadata() throws Exception {
     assertEquals(1, ome.sizeOfPlateList());
   }
 
+  /**
+   * Convert a plate with default options.
+   * The output should be compliant with OME Zarr HCS.
+   */
+  @Test
+  public void testHCSMetadataNoAcquisitions() throws Exception {
+    input = getTestFile("C4-H2-only-no-acqs.xml");
+    assertTool();
+
+    ZarrGroup z = ZarrGroup.open(output);
+
+    int rowCount = 8;
+    int colCount = 12;
+    int fieldCount = 1;
+
+    // Only two rows are filled out with one column each (two Wells total)
+    checkPlateGroupLayout(output, 2, 1, fieldCount, 2, 2);
```

Comment:
This check won't be consistent with lines 907-924.  The idea is to now have basically everything except arrays written for any wells that don't have data, right?  Or have I misunderstood the problem?

Reply:
Discussed earlier today, and I did indeed misunderstand. This PR is meant to follow the example in https://github.com/ome/ngff/blob/0c286908a4fbaceb2c622b21e5a52cc041eea7c0/latest/index.bs#L468, with the added constraint that wells without pixel data *cannot* have a .zgroup.